README.md

```diff
@@ -87,3 +88,5 @@ still set to debug values, which is minutes rather than days.
 
 ## Ownership
 This along with the linked services and repositories is now owned by the [@LBHackney-IT/targeted-services](https://github.com/orgs/LBHackney-IT/teams/targeted-services) team.
+
+-->
```

Comment:
```suggestion

```
Remove comment block to reinstate text (matching the suggestion above). nit: If we're removing the text we should delete it rather than commenting it out.